core/validation/messages.py

```diff
@@ -39,3 +39,6 @@
 )
 # PRE-TRANSFORMATION
 BLANK_SIGN_OFF = "In tab '8 - Review & Sign-Off', cell {cell} is blank but is required."
+
+# POSTCODE
+POSTCODE = "You entered an incorrect postcode. You must enter a postcode like SW1A 2AA"
```

Comment:
Has this gone past design/content?

Reply:
Yep, this came from design through Gemma 👍

Reply:
Can this be updated to match the new content: `You entered an invalid postcode. Enter a full UK postcode, for example SW1A 2AA.`